ui/src/components/NodePartitionTable.js

```diff
@@ -98,15 +104,23 @@ const LoaderContainer = styled.div`
 
 const NodePartitionTable = ({ instanceIP }: { instanceIP: string }) => {
   const theme = useTheme();
+  const { data: alertNF } = useAlerts({
+    alertname: [
+      NODE_FILESYSTEM_SPACE_FILLINGUP,
+      NODE_FILESYSTEM_ALMOST_OUTOF_SPACE,
+      NODE_FILESYSTEM_FILES_FILLINGUP,
+      NODE_FILESYSTEM_ALMOST_OUTOF_FILES,
+    ],
+  });
```

Comment:
I think a filter to retrieve the specific alerts of this instance might be missing

Reply:
Actually, the util function we implemented is OR conditions:
https://github.com/scality/metalk8s/blob/cd742b579be666ef6bf6f0df5f3cc40eac9cd070/ui/src/services/alertUtils.js#L136